packages/attribute-slicer/src/SlicerItem.tmpl.ts

```diff
@@ -30,11 +30,13 @@ import * as d3 from "d3";
 /**
  * Returns an element for the given item
  */
-export default function (item: SlicerItem, sizes: { category: number; value: number }, alignTextLeft: boolean, showValueLabels?: boolean) {
+export default function (item: SlicerItem, sizes: { category: number; value: number }, alignTextLeft: boolean,
+        showValueLabels?: boolean, itemTextColor?: string) {
     "use strict";
     const { match, matchPrefix, matchSuffix, valueSegments, renderedValue } = item;
     const alignStyle = alignTextLeft ? "text-align:left;" : "";
-    const categoryStyle = `display:inline-block;overflow:hidden;max-width:${sizes.category}%;${alignStyle}`;
+    const itemFontColor = itemTextColor ? itemTextColor : "#000";
+    const categoryStyle = `display:inline-block;overflow:hidden;max-width:${sizes.category}%;${alignStyle};color:${itemFontColor}`;
```

Comment:
I forgot, could you add tests to SlicerItem.spec.ts for the three params that was added?